src/actions/nodes.test.js

```diff
@@ -42,24 +43,35 @@ describe('node actions', () => {
       jest.resetModules();
     });
 
+    describe('if meta flag is false', () => {
+      it('should not make any API calls if meta is false', () => {
```

Comment:
This is unnecessary repetition, as it will ultimately read:
```
loadNodeData > if meta flag is false > should not make any API calls if meta is false
```
I'd suggest something like this instead:
```suggestion
    describe('if meta flag is false', () => {
      it('should not make any API calls', () => {
```

Reply:
sounds good!